samples/templates/default-azuredeploy-docker.json

```diff
@@ -123,6 +123,13 @@
                 "description": "Use Cosmos DB free tier."
             }
         },
+        "cosmosDbByokKeyUrl": {
```

Comment:
nit:  similar comment to above.  In this repo, we have been using the phrase/term CMK.  Perhaps this could be cosmosDbCmkUrl or something similar?